devicetypes/smartthings/fibaro-smoke-sensor.src/fibaro-smoke-sensor.groovy

```diff
@@ -481,12 +481,12 @@ private def getTimeOptionValueMap() { [
 		"12 hours"	 : 4320,
 		"18 hours"	 : 6480,
 		"24 hours"	 : 8640,
-		"reports inactive" : 0,
+		"Reports inactive" : 0,
 ]}
 
 private def getNotificationOptionValueMap() { [
 		"disabled" : 0,
```

Comment:
It looks like you are still missing the changes for "disabled" -> "None", "exceeding" -> "Exceeding", "lack" -> "Lack", and "all notifications" -> "All"

Reply:
thanks, good catch